apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -188,8 +197,26 @@ export class ParseEventRequest {
     transactionId,
     discoveredWorkflow?: DiscoverWorkflowOutput | null
   ) {
-    const jobData: IWorkflowDataDto = {
+    const commandArgs = {
```

Comment:
❓ Is it necessary to create a new object?

Reply:
Yes, else the TS keeps yelling for types